cloudwatch_logs_common/CMakeLists.txt

```diff
@@ -1,32 +1,58 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cloudwatch_logs_common)
-set(CLOUDWATCH_LOGS_COMMON_VERSION 1.0.0)
+set(CLOUDWATCH_LOGS_COMMON_VERSION 1.0.0) #todo
```

Comment:
Version should increase (and remove todo)

Reply:
Bumped the minor version and removed the comment.